src/components/App/App.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Coin } from '../../types/types';
+import CoinTable from '../CoinTable';
+import Search from '../Search';
+import Footer from '../Footer';
+import styles from './App.module.scss';
+
+const { app, main } = styles;
+
+type AppProps = Record<string, never>;
+type AppState = {
+  isLoading: boolean;
+  coinList: Coin[];
+};
```

Comment:
I suggest creating a separate file within this folder, e.g.` App.type.ts` and moving the types into this file